### PR TITLE
8.0 de translations

### PR DESCRIPTION
**Pull request by tv-openbig**

### CODE REVIEW

File: project_categ/i18n/de.po [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+#. module: project_categ
+#: model:ir.model,name:project_categ.model_project_task
+msgid "Task"
+msgstr "Aufgabe"
+
+#. module: project_categ
+#: view:project.task:0
+msgid "onchange_project(project_id)"
+msgstr ""
```
Comment (pedrobaeza) [inline project_categ/i18n/de.po]: Fill this with the same value or directly remove it.

File: project_categ/i18n/de.po [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+#. module: project_categ
+#: view:project.task:0
+msgid "{'required':[('task_categ_id','!=',False)]}"
+msgstr ""
```
Comment (pedrobaeza) [inline project_categ/i18n/de.po]: Fill this with the same value or directly remove it.

File: service_desk/i18n/de.po [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+#. module: service_desk
+#: model:ir.model,name:service_desk.model_project_project
+msgid "Project"
+msgstr "Projekt"
+
+#. module: service_desk
+#: model:ir.model,name:service_desk.model_account_analytic_location
+msgid "account.analytic.location"
+msgstr ""
```
Comment (pedrobaeza) [inline service_desk/i18n/de.po]: Fill this with the same value

File: service_desk/i18n/de.po [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+#. module: service_desk
+#: model:ir.model,name:service_desk.model_account_analytic_account
+msgid "Analytic Account"
+msgstr "Vertrag"
+
+#. module: service_desk
+#: view:project.task:0
+msgid "onchange_project(project_id)"
+msgstr ""
```
Comment (pedrobaeza) [inline service_desk/i18n/de.po]: Fill this with the same value or directly remove it.

File: service_desk_issue/i18n/de.po [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+msgid "Project Issue"
+msgstr "Fall"
+
+#. module: service_desk_issue
+#: view:project.issue:0
+msgid ""
+"{'default_project_id': project_id, 'default_name': name, "
+"'default_analytic_account_id': analytic_account_id, 'default_location_id': "
+"location_id}"
+msgstr ""
```
Comment (pedrobaeza) [inline service_desk_issue/i18n/de.po]: Fill this with the same value or directly remove it.

**Comment by dreispt** [comment]
LGTM :+1:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**